packages/node_modules/@webex/plugin-meetings/src/statsAnalyzer/index.js

```diff
@@ -279,12 +362,107 @@ export default class StatsAnalyzer extends EventsScope {
       }
 
       this.statsResults[mediaType][sendrecvType].totalPacketsLost = result.packetsLost;
-      this.packetLossArrayReceiver[mediaType].push(this.statsResults[mediaType][sendrecvType].packetsLost);
       this.statsResults[mediaType][sendrecvType].lastPacketReceivedTimestamp = result.lastPacketReceivedTimestamp;
-      this.statsResults[mediaType][sendrecvType].nackCount = result.nackCount - this.statsResults.internal[mediaType][sendrecvType].nackCount;
+
+
+      // From Thin
+      this.statsResults[mediaType][sendrecvType].totalNackCount = result.nackCount;
+      this.statsResults[mediaType][sendrecvType].totalPliCount = result.pliCount;
+      this.statsResults[mediaType][sendrecvType].framesDecoded = result.framesDecoded;
+      this.statsResults[mediaType][sendrecvType].keyFramesDecoded = result.keyFramesDecoded;
+      this.statsResults[mediaType][sendrecvType].decoderImplementation = result.decoderImplementation;
+      this.statsResults[mediaType][sendrecvType].packetsReceived = result.packetsReceived;
+
+      this.statsResults[mediaType][sendrecvType].fecPacketsDiscarded = result.fecPacketsDiscarded;
+      this.statsResults[mediaType][sendrecvType].packetsReceived = result.packetsReceived;
+      this.statsResults[mediaType][sendrecvType].fecPackets = result.fecPacketsReceived;
+      this.statsResults[mediaType][sendrecvType].totalBytesReceived = result.bytesReceived;
+      this.statsResults[mediaType][sendrecvType].headerBytesReceived = result.headerBytesReceived;
+      this.statsResults[mediaType][sendrecvType].jitter = result.jitter;
     }
   }
 
+  /**
+   * Processes InboundRTP stats result and stores
+   * @private
+   * @param {*} result
+   * @param {*} type
+   * @param {boolean} isSender
+   *
+   * @returns {void}
+   */
+  remotecandidate = (result, type, isSender) => {
```

Comment:
```suggestion
  processInboundRTPResult(result, type, isSender) {
```

Reply:
The function name will be the same i changed the doc

Reply:
Please use the function format at least if you don't want to change the name. I'd prefer the name to be a bit more descriptive as well.